tensornetwork/backends/symmetric/symmetric_backend.py

```diff
@@ -689,3 +689,17 @@ def matmul(self, tensor1: Tensor, tensor2: Tensor):
     if (tensor1.ndim != 2) or (tensor2.ndim != 2):
       raise ValueError("inputs to `matmul` have to be matrices")
     return tensor1 @ tensor2
+
+  def power(self, a: Tensor, b: Tensor) -> Tensor:
+    """
+    Returns the power of tensor a to the value of b.
```

Comment:
The `__pow__`  method of `BlockSparseTensor` does currently not support broadcasting of any type, hence `b` has to be a scalar. Pls  modify the docstring accordingly.

Reply:
I'm a bit confused with this request since this test works fine for me with no failures could you elaborate on what the problem is please?

Reply:
The reason the tests passes is due to a bug in the test (I had actually missed that in the earlier reviewa, see below).
in numpy, `a**b` below does different things depending on how `a` and `b` look like. Generally, numpy applies broadcasting (https://numpy.org/doc/stable/user/basics.broadcasting.html) rules when computing this expression. Typically, this approach is relatively widely used by other libraries as well (e.g. tensorflow or pytorch). Supporting broadcasting for`BlockSparseTensor` is however more complicated because this class has some non-trivial internal structure that needs to be respected  by such operations. Currently we just don't support it. The only case supported by `BlockSparseTensor` is when `b` is a scalar-type object.

Reply:
pls add some test ensuring that `a` and `b` are `BlockSparseTensor`s